src/routes/tools/streamflow/_ExploreData.svelte

```diff
@@ -0,0 +1,264 @@
+<script>
+  import { Loading } from "carbon-components-svelte";
+  import { format } from "d3-format";
+
+  // Helpers
+  import {
+    flattenData,
+    groupDataByYear,
+    formatDataForExport,
+    groupDataByMonth,
+  } from "../_common/helpers";
+  import { getSelectedMonthStrings, getBasinCenter } from "./_helpers";
+  import { serialize } from "~/helpers/utilities";
+  import { DEFAULT_STATION_LAYER } from "./_constants";
+  import { formatMonthlyDataForExport } from "./_data";
+
+  // Components
+  import { Dashboard } from "~/components/tools/Partials";
+  import SettingsPanel from "./_SettingsPanel.svelte";
+  import StreamflowChart from "./_StreamflowChart.svelte";
+  import ChartTitle from "./_ChartTitle.svelte";
+  import StatsPanel from "./_StatsPanel.svelte";
+
+  // Store
+  import {
+    scenarioStore,
+    locationStore,
+    modelsStore,
+    datasetStore,
+    isFetchingStore,
+    dataStore,
+  } from "../_common/stores";
+  import {
+    climvarStore,
+    indicatorStore,
+    selectedMonthsStore,
+    totalAnnual,
+    averageMonthly,
+    selectedPeriodStore,
+    runoffMidpoint,
+  } from "./_store";
+
+  const { location } = locationStore;
+  const { scenario } = scenarioStore;
+  const { indicator } = indicatorStore;
+  const { titles } = datasetStore;
+  const { period } = selectedPeriodStore;
+
+  let data;
+  let dataByDate;
+  let statsData;
+  let showDownload = false;
+  let showShare = false;
+  let showChangeLocation = false;
+  let showLearnMore = false;
+
+  let ChangeStation;
+  let DownloadChart;
+  let ShareLink;
+  let LearnMoreModal;
+
+  let bookmark;
+  let learnMoreProps = {};
+  let metadata;
+  let csvData;
+  let printContainer;
+  let printSkipElements;
+
+  $: chartDescription = $indicator.description;
+
+  $: chartTitle = $location.title;
```

Comment:
Did you check that the title updates when location changes?

Reply:
thanks! missed that one. moved to afterupdate https://github.com/berkeley-gif/caladapt-website-2021/pull/187/commits/3f4b2c1e28cc0268d61fbb6559a5677dc2c3db27